packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache

```diff
@@ -1,24 +1,23 @@
 <div id="main-content" class="card pair-auth">
   <header>
     <h1 id="fxa-pair-auth-allow-header">
-      {{#unsafeTranslate}}Approve new device <small>for %(email)s</small>{{/unsafeTranslate}}
+      {{#t}}Did you just sign in to Firefox?{{/t}}
     </h1>
+    <p>{{#unsafeTranslate}} %(email)s {{/unsafeTranslate}}</p>
   </header>
 
   <section>
     <div class="error"></div>
     <div class="success"></div>
+    <div class="graphic graphic-confirm-pairing" role="img" aria-label="{{#t}}location balloon{{/t}}"></div>
```

Comment:
Q: does this need an `aria-label`?
I'm not 100% certain what a "location balloon" is, or if it will be obvious when read out by a screen reader.

Reply:
A screenreader will read "image, location balloon" which it will read because we have it set to `role="img"`. The alternative is to make this `role="presentation"` instead to tell screenreaders there's not any significance and it can be skipped, but I prefer the former to keep screenreaders' experience more consistent with what sighted users will experience. I personally don't like redundant text since it's unhelpful (e.g. reading the header above it).